forte/ontology_specs/race_qa_onto.json

```diff
@@ -1,7 +1,7 @@
 {
   "ontology_name": "race_multi_choice_qa_ontology",
   "import_paths": [
-    "base_ontology.json"
+    "ft.onto.base_ontology"
```

Comment:
The key name `import_paths` is not very intuitive when we use packages here. Maybe we should separate these to another key, `import` ?

Reply:
Done.